### PR TITLE
Optee os 3.20+fio testing

### PR DESCRIPTION
**Pull request by igoropaniuk**

### CODE REVIEW

File: meta-lmp-base/recipes-security/optee/optee-os-fio.inc [hunk 1]
```diff
@@ -34,6 +34,7 @@ EXTRA_OEMAKE += " \
     CROSS_COMPILE_ta_${OPTEE_ARCH}=${HOST_PREFIX} \
     CFG_WERROR=y DEBUG=0 CFG_OPTEE_REVISION_EXTRA=+fio \
     CFG_TEE_CORE_LOG_LEVEL=2 CFG_TEE_TA_LOG_LEVEL=2 \
+    CFG_ENABLE_EMBEDDED_TESTS=y \
```
Comment (ricardosalveti) [inline meta-lmp-base/recipes-security/optee/optee-os-fio.inc:37]: Good for testing but we need to remember to drop before merging the final pr.

**Review by angolini — APPROVED** [review]
Assuming the drop of 3.18 is intentional, the PR LGTM

**Review by MrCry0 — APPROVED** [review]
LGTM

**Review by quaresmajose — APPROVED** [review]

**Comment by ricardosalveti** [comment]
Added the se050 fallback and a bump to the latest from our repo at https://github.com/foundriesio/meta-lmp/pull/1059 (taking this pr as reference). Will close this one in favor of the new one.

**Pull request closed without merging**